app/features/analysis/calculate/OneRMCalculateActions.js

```diff
@@ -167,12 +168,24 @@ const logCalculate1RMAnalytics = (state, exercise, numIncludeTags, numExcludeTag
         slope: slope,
         min_weight: minWeight,
         max_weight: maxWeight,
+        min_weight_in_lb: weightInLB(metric, minWeight),
+        max_weight_in_lb: weightInLB(metric, maxWeight),
+        weight_lb_range: weightInLB(metric, (maxWeight - minWeight)),
         min_velocity: minVelocity,
         max_velocity: maxVelocity,
+        velocity_range: maxVelocity - minVelocity,
         metric: metric
     }, state);
 };
 
+const weightInLB = (metric, weight) => {
+    if (metric === 'lbs') {
+        return weight;
+    } else if (metric === 'kgs') {
+        return weight * 2.20462262;
+    }
+}
```

Comment:
- [ ] Make a new utils class for this, then have SetUtils use that weight util function

Reply:
Would SetUtils be the appropriate place for this? It does not pertain exclusively to sets.